index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
```

Comment:
- [ ] As required for this project, I looked for the `.gitignore` file that contains the `test.md` file and the `/node_modules` folder name but could not find it. Essentially, what `.gitignore` files does is that it prevents you from pushing unnecessary code to your GitHub repository. One of such file/folder that should never be pushed to GitHub is the `node_module` folder. The test.md is a requirement for this project. Kindly fix this.